### PR TITLE
Some functions had been moved away from include/items.php

### PR DESCRIPTION
**Pull request by annando**

### CODE REVIEW

File: src/Protocol/DFRN.php [hunk 1]
```diff
@@ -3089,4 +3089,73 @@ private static function constructVerb(array $item)
...
+	/**
+	 * This function returns true if $update has an edited timestamp newer
+	 * than $existing, i.e. $update contains new data which should override
+	 * what's already there.  If there is no timestamp yet, the update is
+	 * assumed to be newer.  If the update has no timestamp, the existing
+	 * item is assumed to be up-to-date.  If the timestamps are equal it
+	 * assumes the update has been seen before and should be ignored.
+	 *
+	 */
+	private static function editedTimestampIsNewer($existing, $update)
```
Comment (MrPetovan) [inline src/Protocol/DFRN.php]: Please rename to `isEditedTimestampNewer`.

File: src/Protocol/DFRN.php [hunk 2]
```diff
@@ -3089,4 +3089,73 @@ private static function constructVerb(array $item)
 		}
 		return ACTIVITY_POST;
 	}
+
+	private static function tgroupCheck($uid, $item)
```
Comment (MrPetovan) [inline src/Protocol/DFRN.php]: What's a `tgroup`? What does this function?

Comment (annando) [reply src/Protocol/DFRN.php]: Don't know. It has something to do with forums.

File: src/Protocol/DFRN.php (hunk 1 above)
Comment (annando) [reply src/Protocol/DFRN.php]: Done

**Pull request merged**